src/middlewared/middlewared/plugins/config.py

```diff
@@ -143,7 +144,11 @@ def upload_impl(self, file_or_tar, is_tar_file=False):
             if found_db_file is None:
                 raise CallError('Neither a valid tar or TrueNAS database file was provided.')
 
-            p = subprocess.run(['migrate', str(found_db_file.absolute())], capture_output=True, text=True)
+            p = subprocess.run([
+                'migrate',
+                str(found_db_file.absolute()),
+                f'{temp_dir}/{PWENC_SECRET_NAME}',
```

Comment:
What happens when someone uploads just a backup of the database and doesn't have a secret seed?

Reply:
No error occurs, non-existent pwenc secret just results in not being able to decrypt anything and everything works as usual